plugin.json

```
@@ -6,7 +6,7 @@
     "support_url": "https://github.com/mattermost/mattermost-plugin-zoom/issues",
     "release_notes_url": "https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.5.1",
```

FYI we always have to change this value too
```suggestion
    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.7.0",
```

---

Noted, thanks